### PR TITLE
Clarify z-wave polling.

### PR DESCRIPTION
**Pull request by sveip**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_docs/z-wave/devices.markdown [hunk 1]
```diff
@@ -61,7 +61,9 @@ As long as your device lists Hail or Association in its Controlled Command Class
...
 
-For example, with `polling_interval=60000` (which is the default) if you have 10 devices that are being polled, if a device can receive and acknowledge the poll within one second, then it will take 10 seconds to complete the polling list, which leaves 50 seconds left for normal traffic. The more devices you poll, and the shorter the interval, the less bandwidth that's available for normal traffic.
+For example, with `polling_interval=60000` (which is the default) if you have 10 devices that are being polled at every interval, and each polling takes one secound (request/response round trip), then it will take 10 seconds to complete the polling list. This only leaves 50 seconds left for normal traffic. The more devices you poll, and the shorter the interval, the less bandwidth that's available for normal traffic.
+
+Polling is by default disabled and needs to be anebled per device. You can also specify the interval by setting the *polling intensity* of the device. See the [Node Management](/docs/z-wave/control-panel#z-wave-node-management) docs for details.
```
Comment (klaasnicolaas) [inline source/_docs/z-wave/devices.markdown]:
```suggestion
Polling is by default disabled and needs to be enabled per device. You can also specify the interval by setting the *polling intensity* of the device. See the [Node Management](/docs/z-wave/control-panel#z-wave-node-management) docs for details.
```

Are you sure that polling is by default disabled?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_docs/z-wave/devices.markdown (hunk 1 above)
Comment (sveip) [reply source/_docs/z-wave/devices.markdown]: I wasn't, but Tinkerer told me it is. 

I checked the source now, and it seems like that polling is in fact disabled if `polling_intensity` isn't set as far as I understand:

```python
        polling_intensity = convert(
            node_config.get(CONF_POLLING_INTENSITY), int)
        if polling_intensity:
            self.primary.enable_poll(polling_intensity)
```

Comment (sveip) [reply source/_docs/z-wave/devices.markdown]: @klaasnicolaas, by investigating this I also saw that the doc on the `polling_intensity` in the linked article is also a bit lacking, so maybe this whole subject should be looked at and potentially merged and corrected by someone that has deeper knowledge?

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_docs/z-wave/devices.markdown (hunk 1 above)
Comment (DubhAd) [reply source/_docs/z-wave/devices.markdown]: > I wasn't, but Tinkerer told me it is.

More that _it isn't enabled on devices_ by default, not that it's disabled _totally_ by default

File: source/_docs/z-wave/devices.markdown [hunk 2]
```diff
@@ -61,7 +61,9 @@ As long as your device lists Hail or Association in its Controlled Command Class
...
 
-For example, with `polling_interval=60000` (which is the default) if you have 10 devices that are being polled, if a device can receive and acknowledge the poll within one second, then it will take 10 seconds to complete the polling list, which leaves 50 seconds left for normal traffic. The more devices you poll, and the shorter the interval, the less bandwidth that's available for normal traffic.
+For example, with `polling_interval=60000` (which is the default) if you have 10 devices that are being polled at every interval, and each polling takes one secound (request/response round trip), then it will take 10 seconds to complete the polling list. This only leaves 50 seconds left for normal traffic. The more devices you poll, and the shorter the interval, the less bandwidth that's available for normal traffic.
+
+Polling is by default disabled and needs to be enabled per device. You can also specify the interval by setting the *polling intensity* of the device. See the [Node Management](/docs/z-wave/control-panel#z-wave-node-management) docs for details.
```
Comment (DubhAd) [inline source/_docs/z-wave/devices.markdown]: How about:

> Polling needs to be enabled per device, and that is controlled by the *polling intensity* of the device. See the [Node Management](/docs/z-wave/control-panel#z-wave-node-management) docs for details.

**Review by DubhAd — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
I have adjusted the last line, please give your opinion. 

I miss for the lack of clarity where / how you can turn on the polling, if I understand correctly you do that by means of the polling intensity, 0 = off and > 0 is on?

**Comment by turbokongen** [comment]
Polling is disabled if not specified in config. If Polling is changed in the ui, 0 is off anything else is on.

**Review by DubhAd — APPROVED** [review]
👍

**Pull request merged**